src/dashboard/apps/renewable/fixtures/renewable.py

```diff
@@ -0,0 +1,29 @@
+"""Dashboard renewable dev fixture."""
+
+import random
+
+from apps.core.factories import DeliveryPointFactory, StationFactory
+from apps.core.models import DeliveryPoint
+
+from ..factories import RenewableFactory
+
+
+def seed_renewable():
+    """Creates development fixtures for renewable management system."""
+    if not DeliveryPoint.objects.exists():
```

Comment:
IMO you should always create new deliverypoint, or else quit if you prefer:

```suggestion
    if DeliveryPoint.objects.exists() and not force:
        return
```

Reply:
The idea is to use the `delivery_points` that were created during the `seed_consents` and only create some if the `seed_consent` is not passed

Reply:
In this case, why not creating an explicit dependency between the commands instead?

```suggestion
    if not DeliveryPoint.objects.exists():
        raise(DatabaseError("We expect delivery points, run seed consents first."))
```